pyart/graph/gridmapdisplay.py

```diff
@@ -107,38 +109,28 @@ def plot_basemap(self, lat_lines=None, lon_lines=None,
             Axis to add the basemap to, if None the current axis is used.
 
         """
+        # make basemap
+        self._make_basemap(resolution, area_thresh, auto_range,
+                           min_lon, max_lon, min_lat, max_lat, ax)
+
         # parse the parameters
-        ax = self._parse_ax(ax)
         if lat_lines is None:
             lat_lines = np.arange(30, 46, 1)
         if lon_lines is None:
             lon_lines = np.arange(-110, -75, 1)
 
-        # determine map region
-        if auto_range:
-            max_lat = self.grid_lats.max()
-            max_lon = self.grid_lons.max()
-            min_lat = self.grid_lats.min()
-            min_lon = self.grid_lons.min()
-
-        if self.debug:
-            print("Maximum latitude: ", max_lat)
-            print("Maximum longitude: ", max_lon)
-            print("Minimum latitude: ", min_lat)
-            print("Minimum longitute: ", min_lon)
-
-        # plot the basemap
-        basemap = Basemap(llcrnrlon=min_lon, llcrnrlat=min_lat,
-                          urcrnrlon=max_lon, urcrnrlat=max_lat,
-                          projection='mill', area_thresh=area_thresh,
-                          resolution=resolution, ax=ax)
-        basemap.drawcoastlines(linewidth=1.25)
-        basemap.drawstates()
-        basemap.drawparallels(lat_lines, labels=[True, False, False, False])
-        basemap.drawmeridians(lon_lines, labels=[False, False, False, True])
-        self.basemap = basemap
-
-    def plot_grid(self, field, level=0, vmin=None, vmax=None, cmap='jet'):
+        self.basemap.drawcoastlines(linewidth=1.25)
+        self.basemap.drawstates()
+        self.basemap.drawparallels(lat_lines,
+                                   labels=[True, False, False, False])
+        self.basemap.drawmeridians(lon_lines,
+                                   labels=[False, False, False, True])
+
+    def plot_grid(self, field, level=0, vmin=None, vmax=None, cmap='jet',
+                  mask_outside=True, title=None, title_flag=True,
```

Comment:
the default is `True` in [ppi](https://github.com/ARM-DOE/pyart/blob/master/pyart/graph/radardisplay.py#L265) and [rhi](https://github.com/ARM-DOE/pyart/blob/master/pyart/graph/radardisplay.py#L365) !?! But if it is to be `False` I have no objection